contrib/epee/include/net/http_protocol_handler.inl

```diff
@@ -328,8 +328,15 @@ namespace net_utils
 	inline bool analize_http_method(const boost::smatch& result, http::http_method& method, int& http_ver_major, int& http_ver_minor)
 	{
 		CHECK_AND_ASSERT_MES(result[0].matched, false, "simple_http_connection_handler::analize_http_method() assert failed...");
-		http_ver_major = boost::lexical_cast<int>(result[11]);
-		http_ver_minor = boost::lexical_cast<int>(result[12]);
+		try
```

Comment:
```c++
if (!boost::conversion::try_lexical_convert(result[11], http_ver_major))
    return false;
```
Is available in Boost 1.57+. Internally, `boost::lexical_cast` is doing this call anyway.

Reply:
Thank, looks like the min boost version for monero is now 1.58, I'll change it.